docs/modules/getting_started/pages/flow-deep-dive.adoc

```diff
@@ -0,0 +1,42 @@
+:imagesdir: ../assets/images
+[[deep-dive-dash]]
+= Flow Deep-Dive dashboard
+
+The {page-component-title} plugin includes a pre-designed flow deep-dive dashboard that displays flow data.  
+The dashboard can help with quality of service (QoS) monitoring and includes panels that display data in the following categories:
+
+* flow interface statistics (throughput and data use by application)
+* SNMP interface statistics (interface throughput and errors/discards)
+* conversations (throughput and data use by conversation)
+* hosts (throughput and data use by host)
+* DCSPs (throughput and data use by Differentiated Services Code Points)
+
+== QoS options
+
+The dashboard exposes nodes and interfaces with flows, that you can filter on in the *Node* and *Interface* drop-down lists.  
+
+You can also filter flows traffic by ToS/DSCP and ECN (explicit congestion notification) for nodes and interfaces with these values. 
+
+Select a DSCP value (or combination of values) from the list for the QoS you want to visualize (class selector, IP precedence, assured forwarding, expedited forwarding). 
+
+The ECN drop-down lets you further filter on this value to see which flows use ECN-capable transports, and congestion reports for them.  
```

Comment:
There is no ECN drop-down. (I would have liked to have it but that is another story...)

Reply:
```suggestion
```